dbt/models/eia860/core_eia860__scd_generators_energy_storage/schema.yml

```diff
@@ -4,6 +4,10 @@ sources:
     tables:
       - name: core_eia860__scd_generators_energy_storage
         data_tests:
+          - expect_columns_not_all_null:
+              exclude_columns:
+                - storage_technology_code_3
+                - storage_technology_code_4
```

Comment:
These seem to be placeholders for a future in which there could be more than 2 energy storage techs deployed in a place, so it seems okay that they're all null for now. But it would be good if we could automatically catch when they are no longer all null in the future. Where is the right place to do that and how?